website/docs/addon_nuke_artist.md

```diff
@@ -395,3 +395,14 @@ If your Pyblish dialog fails on Validate Version, you might be trying to publish
 Or maybe you accidentally copied write node from different shot to your current one. Check the write publishes on the left side of the Pyblish dialog. Typically you publish only one write. Locate and delete the stray write from other shot.
 
 <iframe width="512px" height="288px" src="https://www.youtube.com/embed/Ic9z4gKnHAA" frameborder="0" modestbranding="1" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="1"></iframe>
+
+### Inconsistent FBX Camera loading
+
+If you load a FBX camera from Nuke, you might get some error with incomplete loading especially if your workfile already contains ABC data. Nuke does support [Importing Camera from FBX files](https://learn.foundry.com/nuke/content/comp_environment/3d_compositing/importing_fbx_cameras.html), but the Camera node sometimes fail to load the FBX file.
+
+This is because ABC and FBX seems to share a common loading context and having both in the same workfile conflicts.
+
+Fix this by reloading the FBX file manually then ensure the "node name" is set to the Camera.
+("Producer Perspective", "Producer Top", "Producer Bottom"... are default FBX view and automatically added by Nuke).
```

Comment:
who doesn't love ai? 
anyways, I'm not sure what is the best place to put the info. 

```suggestion
When loading an FBX camera, you may encounter incomplete loading error, particularly if your wokfile already contains Alembic (ABC) data. Although Nuke supports [FBX camera imports](https://learn.foundry.com/nuke/content/comp_environment/3d_compositing/importing_fbx_cameras.html), the Camera node can sometimes fail to load the FBX file properly.

This issue often arises because ABC and FBX formats seem to share a common loading context, which can lead to conflicts when both are present in the same workfile.

**To resolve this:**
1. Try manually reloading the FBX file. 
2. Make sure the `node name` is set to the Camera. 

:::info Default FBX Views
views like "Producer Perspective," "Producer Top," and "Producer Bottom" are default FBX views that Nuke adds automatically.
:::
```

Also, here's how the suggestion above looks on my side
![image](https://github.com/user-attachments/assets/d6a98ba3-e6d6-475f-ad51-dcc963c6862b)

Reply:
Love it !
Nuke addons changes were merged last week, so I think this one is ready to go.